k8s/src/test/scala/io/buoyant/k8s/EndpointsNamerTest.scala

```diff
@@ -1106,6 +1106,88 @@ class EndpointsNamerTest extends FunSuite with Awaits {
     }
   }
 
+  test("initial get returns unexpected status") {
+    @volatile var i = 0
+    val service = Service.mk[Request, Response] {
+      case req if req.uri == "/api/v1/namespaces/srv/endpoints/sessions" && i == 0 =>
+        val rsp = Response()
+        rsp.status = http.Status.NotAcceptable
```

Comment:
would be cute to use ScalaCheck here & generate an arbitrary status code...definitely not necessary though.